addOns/exportreport/src/main/java/org/zaproxy/zap/extension/exportreport/export/ReportExportPDF.java

```diff
@@ -0,0 +1,1111 @@
+            throws IOException {
+        if (!imagePath.isEmpty()) {
+
+            // create the image
+            PDXObjectImage image = null;
+            BufferedImage awtImage = ImageIO.read(new File(imagePath));
+            image = new PDPixelMap(document, awtImage);
+
+            // and scale it
+            if (scalePercent != 0) {
+                int newWidth = (int) (awtImage.getWidth() * (scalePercent / 100));
+                int newHeight = (int) (awtImage.getHeight() * (scalePercent / 100));
+                BufferedImage resized = new BufferedImage(newWidth, newHeight, awtImage.getType());
+                Graphics2D g = resized.createGraphics();
+                g.setRenderingHint(
+                        RenderingHints.KEY_INTERPOLATION,
+                        RenderingHints.VALUE_INTERPOLATION_BILINEAR);
+                g.drawImage(
+                        awtImage,
+                        0,
+                        0,
+                        newWidth,
+                        newHeight,
+                        0,
+                        0,
+                        awtImage.getWidth(),
+                        awtImage.getHeight(),
+                        null);
+                g.dispose();
+                image = new PDPixelMap(document, resized);
+            }
+
+            // centre align the image.  Note that the page dimension units here are in points.
+            float pageWidthPoints = page.getMediaBox().getWidth();
+            float imageWidthInPoints = image.getWidth();
+
+            // apparently this must be created after the image is created, for some reason..
+            PDPageContentStream contentStream = new PDPageContentStream(document, page, true, true);
+
+            contentStream.drawImage(
+                    image,
+                    (pageWidthPoints / 2) - (imageWidthInPoints / 2),
+                    (float) textInsertionPoint.getY() - image.getHeight());
+            contentStream.close();
+
+            // update the text insertion point after drawing an image
+            textInsertionPoint =
+                    new Point2D.Float(
+                            marginPoints, (float) (textInsertionPoint.getY() - image.getHeight()));
+        }
+
+        return textInsertionPoint;
+    }
+
+    /**
+     * get content for the named alert category, using the key provided, and the default value
+     * provided.
+     *
+     * @param pluginId
+     * @param key
+     * @param contentDefault
+     * @param extensionExport
+     * @return
+     */
+    private static String getFieldAlertProperty(
+            int pluginId,
+            String key,
+            String contentDefault,
+            ExtensionExportReport extensionExport) {
+        if (key.contains("risk") || key.contains("reliability")) {
+            return getMessage(
+                    extensionExport, "alertreport.export.pluginid." + key, contentDefault);
```

Comment:
Should be changed to `exportreport`, also worth adding a task tag for further review/changes.